src/process/redux/store.js

```diff
@@ -6,6 +6,7 @@ import { combineReducers, configureStore } from '@reduxjs/toolkit';
 import userReducer from 'process/slices/userSlice';
 import eventsReducer from 'process/slices/eventsSlice';
 import driversReducer from 'process/slices/driversSlice';
+import transientReducer from 'process/slices/transientSlice';
 import constructorsReducer from 'process/slices/constructorsSlice';
```

Comment:
I see those are becoming to many, you can create and index.js file in /slices and then you can import all of those slices in there, then export them so in this file (store.js) you can import the from the same place.

Reply:
done